pom.xml

```diff
@@ -277,6 +277,10 @@
                                             <project.groupId>${project.groupId}</project.groupId>
                                         </systemProperties>
                                     </defaultTestConfig>
+                                    <!--
+                                    Build step 'org.kie.kogito.infinispan.quarkus.deployment.KogitoAddOnPersistenceInfinispanProcessor#verifyCapabilities' does not produce any build item and thus will never get executed
+                                    -->
```

Comment:
@evacchi @cristianonicolai @mbiarnes this commit will have to be reverted when we upgrade Kogito.

Reply:
/cc @radtriste

Reply:
oki
cc @mbiarnes

Reply:
This would be for Kogito 1.27 then